src/models/secretCodes.js

```diff
@@ -0,0 +1,29 @@
+import { BaseModel } from './base';
+
+class SecretCodeModel extends BaseModel {
+  constructor() {
+    super('secret_codes');
+  }
+
+  async addSecretCode(values) {
```

Comment:
Já que só funciona com 3 valores, será que nao ficava melhor receber eles mais explicitamente?
```suggestion
  async addSecretCode(code, userId, expirationDate) {
```
Aí lá embaixo fica:
```js
return this.pool.query(query, [code, userId, expirationDate]);
```